yarn.lock

```diff
@@ -228,7 +228,7 @@
     "@agoric/assert" "^0.6.0"
     "@endo/marshal" "^0.8.5"
 
-"@agoric/smart-wallet@^0.5.3":
+"@agoric/smart-wallet@0.5.3", "@agoric/smart-wallet@^0.5.3":
```

Comment:
did you mean to pin the version in package.json?

Reply:
Removed